env.yml

```diff
@@ -0,0 +1,40 @@
+name: VPN
+channels:
+  - conda-forge
+dependencies:
+  - clang=14.0.6
+  - clangxx=14.0.6
+  - lld=14.0.6
+  - llvmdev=14.0.6
+  - llvm=14.0.6
+  - python=3.9
+  - nodejs=16.13.*
+  - pip=22.3.1
+  - rust=1.66
+  - go=1.18
```

Comment:
Note - this is an updade of go -> we had `go1.17.6.
On windows since 2.11.2 we run  1.18.